packages/client-sdk-nodejs/src/auth-client.ts

```diff
@@ -14,36 +9,4 @@ export class AuthClient extends AbstractAuthClient implements IAuthClient {
 
     super({createAuthClient: () => authClient});
   }
-
-  /**
-   * Gets a api token, refresh token given a valid session token.
-   *
-   * @param {string} controlEndpoint - Endpoint for control plane.
-   * @param {string} token - The token to allow access for generation of api tokens.
-   * @param {string} expiresIn - How long the token is valid for in epoch timestamp.
-   * @returns {Promise<GenerateAuthToken.Response>} -
-   * {@link GenerateAuthToken.Success} containing the api token, refresh token, origin and epoch timestamp when token expires.
-   * If the token never expires, then no refresh token will be returned and expires at timestamp will be infinite.
-   * {@link GenerateAuthToken.Error} on failure.
-   */
-  public async generateAuthToken(
```

Comment:
i changed these to just properties instead of functions, based on some feedback from @allenheltondev in the past.